autokeras/tasks/text.py

```diff
@@ -36,16 +42,16 @@ class TextClassifier(SupervisedTextPipeline):
     """
 
     def __init__(self,
-                 num_classes=None,
-                 multi_label=False,
-                 loss=None,
-                 metrics=None,
-                 name='text_classifier',
-                 max_trials=100,
-                 directory=None,
-                 objective='val_loss',
-                 overwrite=True,
-                 seed=None):
+                 num_classes: Optional[int] = None,
+                 multi_label: bool = False,
+                 loss: Callable = None,
+                 metrics: Optional[List[Callable]] = None,
```

Comment:
I think loss and metrics should be more specific.
The callable should be keras loss and keras metric.
Loss can also be a string.
Metric can also be a list of strings.
They can also be dictionaries.
https://keras.io/models/model/#compile

Reply:
Nice! This will clarify it for people reading the docs :)